### PR TITLE
Finished translation of React Developer Tools page

### PR DESCRIPTION
**Pull request by alinkedd**
- Finished translation of React Developer Tools page
- Added two words (app - `застосунок`, browser - `браузер`) to translation guide.
  - In case translation for an app as `застосунок` will be approved, then other pages should have `додаток` replaced in the same context. Here are some thoughts that leaded to my decision:
    - `застосунок` is the main entry point for an [app](https://uk.wikipedia.org/wiki/%D0%97%D0%B0%D1%81%D1%82%D0%BE%D1%81%D1%83%D0%BD%D0%BE%D0%BA#%D0%A2%D0%B5%D1%80%D0%BC%D1%96%D0%BD_%D0%B2_%D1%83%D0%BA%D1%80%D0%B0%D1%97%D0%BD%D1%81%D1%8C%D0%BA%D1%96%D0%B9_%D0%BC%D0%BE%D0%B2%D1%96) article at ukrainian wikipedia, while `додаток` is an entry point for [`add-on`](https://uk.wikipedia.org/wiki/%D0%94%D0%BE%D0%B4%D0%B0%D1%82%D0%BE%D0%BA_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%B0))-like tools. Also the most neutral `прикладна програма` is a bit long and haven't been used widely in this context. `аплікація` has received completely different meaning.
    - [online corrector](https://onlinecorrector.com.ua/uk/%D0%B7%D0%B0%D1%81%D1%82%D0%BE%D1%81%D1%83%D0%BD%D0%BE%D0%BA-%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%B0/) tells to use `застосунок`
    - more and more people are telling to use `застосунок`, 'cause `додаток` was wrongly adopted from russian homonym `приложение` in context of contracts/law/documentation. (приложить/приложение - додати/додаток, прилагать/приложение - застосувати/застосунок)
  - `браузер` is added just to standardize its usage throughout the project and eliminate other variants like `бравзер`, `переглядач`, or `вебпереглядач`

### CODE REVIEW

File: src/content/learn/react-developer-tools.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
 ---
-title: React Developer Tools
+title: Інструменти розробки React
 ---
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Використовуйте інструменти розробки React (_React Developer Tools_) для інспектування [компонентів](/learn/your-first-component) React, редагування їх [пропсів](/learn/passing-props-to-a-component) і [стану](/learn/state-a-components-memory), а також для виявлення проблем з продуктивністю.
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:7]:
```suggestion
Використовуйте інструменти React розробника (_React Developer Tools_) для інспектування React [компонентів](/learn/your-first-component), редагування їх [пропсів](/learn/passing-props-to-a-component) і [стану](/learn/state-a-components-memory), а також для виявлення проблем з продуктивністю.
```

File: src/content/learn/react-developer-tools.md [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Як встановити інструменти розробки React
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:13]:
```suggestion
* Як встановити інструменти React розробника
```

File: src/content/learn/react-developer-tools.md [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
 
 </YouWillLearn>
 
-## Browser extension {/*browser-extension*/}
+## Розширення браузера {/*browser-extension*/}
 
-The easiest way to debug websites built with React is to install the React Developer Tools browser extension. It is available for several popular browsers:
+Найпростіший спосіб налагодження вебсайтів, створених за допомогою React — встановити розширення браузера "Інструменти розробки React" (_React Developer Tools_). Воно доступне для декількох популярних браузерів:
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:19]:
```suggestion
Найпростіший спосіб налагодження вебсайтів, створених за допомогою React — встановити розширення браузера "Інструменти React розробника" (_React Developer Tools_). Воно доступне для декількох популярних браузерів:
```

File: src/content/learn/react-developer-tools.md [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
 
-* [Install for **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
-* [Install for **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
-* [Install for **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
+* [Встановити для **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=uk)
+* [Встановити для **Firefox**](https://addons.mozilla.org/uk/firefox/addon/react-devtools/)
+* [Встановити для **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil?hl=uk)
 
-Now, if you visit a website **built with React,** you will see the _Components_ and _Profiler_ panels.
+Тепер під час відвідування вебсайту, **створеного за допомогою React**, ви побачите вкладки _Components_ і _Profiler_.
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:25]:
```suggestion
Тепер, якщо ви відвідаєте вебсайт, **який створений за допомогою React**, ви побачите вкладки _Components_ і _Profiler_.
```

File: src/content/learn/react-developer-tools.md [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
 
-![React Developer Tools extension](/images/docs/react-devtools-extension.png)
+![Розширення 'Інструменти розробки React'](/images/docs/react-devtools-extension.png)
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:27]:
```suggestion
![Розширення 'Інструменти React розробника'](/images/docs/react-devtools-extension.png)
```

File: src/content/learn/react-developer-tools.md [hunk 6]
```diff
@@ -36,26 +36,26 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal:
+Далі відкрийте інструменти розробки з терміналу:
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:39]:
```suggestion
Далі відкрийте інструменти розробника з терміналу:
```

File: src/content/learn/react-developer-tools.md [hunk 7]
```diff
@@ -36,26 +36,26 @@ yarn global add react-devtools
...
 ```bash
 react-devtools
 ```
 
-Then connect your website by adding the following `<script>` tag to the beginning of your website's `<head>`:
+Потім під'єднайте інструменти, вказавши наступний тег `<script>` на початку елементу `<head>` у коді вашого вебсайту:
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:44]:
```suggestion
Потім під'єднайте ваш вебсайт, вказавши наступний тег `<script>` на початку елементу `<head>` у коді вашого вебсайту:
```

File: src/content/learn/react-developer-tools.md [hunk 8]
```diff
@@ -36,26 +36,26 @@ yarn global add react-devtools
...
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
 
-Reload your website in the browser now to view it in developer tools.
+Перезавантажте вебсайт у браузері, щоб переглянути його в інструментах розробки.
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:51]:
```suggestion
Перезавантажте вебсайт у браузері, щоб переглянути його в інструментах розробника.
```

File: src/content/learn/react-developer-tools.md [hunk 9]
```diff
@@ -36,26 +36,26 @@ yarn global add react-devtools
...
 
-![React Developer Tools standalone](/images/docs/react-devtools-standalone.png)
+![Автономні 'Інструменти розробки React'](/images/docs/react-devtools-standalone.png)
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:53]:
```suggestion
![Автономні 'Інструменти React розробника'](/images/docs/react-devtools-standalone.png)
```

File: src/content/learn/react-developer-tools.md [hunk 10]
```diff
@@ -36,26 +36,26 @@ yarn global add react-devtools
...
 
-## Mobile (React Native) {/*mobile-react-native*/}
-React Developer Tools can be used to inspect apps built with [React Native](https://reactnative.dev/) as well.
+## Мобільний застосунок (React Native) {/*mobile-react-native*/}
+Інструменти розробки React також можна використовувати для інспектування застосунків, створених за допомогою [React Native](https://reactnative.dev/).
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:56]:
```suggestion
Інструменти React розробника також можна використовувати для інспектування застосунків, створених за допомогою [React Native](https://reactnative.dev/).
```

File: src/content/learn/react-developer-tools.md [hunk 11]
```diff
@@ -64,13 +64,13 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal.
+Далі відкрийте інструменти розробки з терміналу:
```
Comment (bkatsevych) [inline src/content/learn/react-developer-tools.md:67]:
```suggestion
Далі відкрийте інструменти розробника з терміналу:
```

**Review by bkatsevych — APPROVED** [review]

File: src/content/learn/react-developer-tools.md [hunk 12]
```diff
@@ -1,33 +1,33 @@
 ---
-title: React Developer Tools
+title: Інструменти розробки React
```
[the remark this answers is not in the document]
Comment (alinkedd) [reply src/content/learn/react-developer-tools.md]: our guide contain translation of Developer Tools  - `Інструменти розробки`, thus I translated based on that

File: src/content/learn/react-developer-tools.md (hunk 4 above)
Comment (alinkedd) [reply src/content/learn/react-developer-tools.md]: imho, that word-by-word translation is not so good and appropriate in Ukrainian

**Pull request merged**